guided-match/guided-match-service.yaml

```diff
@@ -34,6 +34,68 @@ tags:
 
 paths:
 
+  /search-journeys:
+    get:
+      summary: Get list of journeys which match the searchTerm
+      description: |
+        Given a searchTerm, looks up which journeys match (using fuzzy search) and returns the details needed toprovide the journey selection screen and/or enter the journey

```

Comment:
Teeny point - but missing space in `toprovide`

Reply:
corrected